src/searchApiClient.ts

```diff
@@ -0,0 +1,8 @@
+export function fetchSearchResultsFromAPI(searchTerm) {
+  // Note: this API seems a bit unstable and sometimes it takes a long time to respond, especially
+  // for broad searches
+
+  return fetch(
+    `https://cat-api-bjoerge.sanity-io1.now.sh/cats?query=${searchTerm}`
+  ).then((res) => res.json())
```

Comment:
Fetch doesn't throw on 400 & 500 errors so this will break down the line. You should check if response is ok, else throw an error.